pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/fraser/barnepensjon/Lover.kt

```diff
@@ -1,18 +1,21 @@
 package no.nav.pensjon.etterlatte.maler.fraser.barnepensjon
 
 import no.nav.pensjon.brev.template.LangBokmal
+import no.nav.pensjon.brev.template.LangBokmalNynorskEnglish
 import no.nav.pensjon.brev.template.Language
 import no.nav.pensjon.brev.template.OutlinePhrase
 import no.nav.pensjon.brev.template.dsl.OutlineOnlyScope
 import no.nav.pensjon.brev.template.dsl.text
 
 object Lover {
 
-    object Folketrygdloven18ogforvaltningsloven35_1_c : OutlinePhrase<LangBokmal>() {
-        override fun OutlineOnlyScope<LangBokmal, Unit>.template() {
+    object Folketrygdloven18ogforvaltningsloven35_1_c : OutlinePhrase<LangBokmalNynorskEnglish>() {
+        override fun OutlineOnlyScope<LangBokmalNynorskEnglish, Unit>.template() {
             paragraph {
                 text(
                     Language.Bokmal to "Vedtaket er gjort etter bestemmelsene i folketrygdloven kapittel 18 og forvaltningsloven § 35 første ledd bokstav c.",
+                    Language.Nynorsk to "",
```

Comment:
Dette virker uheldig, spesielt når dette ikke er en redigerbar komponent.

Reply:
Alt med både nynorsk og engelsk er høgst uferdig og ikkje tenkt at vi skal bruke enno. Ventar på omsetjing der, har mest definert det her for at det skal kompilere.